utils/utils.go

```diff
@@ -1083,3 +1085,42 @@ func ServiceAccountPullSecretExists(ba common.BaseComponent, client client.Clien
 	}
 	return nil
 }
+
+// Get security context from CR and apply customization to default settings
+func getSecurityContext(ba common.BaseComponent) *corev1.SecurityContext {
+	rcSecurityContext := ba.GetSecurityContext()
```

Comment:
Suggestion: rcSecurityContext --> baSecurityContext 
since this util method is applicable to other operators as well